eagle-assembly/src/main/README.md

```diff
@@ -91,6 +91,9 @@ Sandbox Starter
 		
 * check eagle UI <http://127.0.0.1:9099/eagle-service>
 
+If you get a 404 Error when trying to access the UI, add port 9099 to "Settings->Network->Advanced->Port Forwarding" in VirtualBox.
+(Instruction #4 in https://eagle.incubator.apache.org/docs/quick-start.html)
```

Comment:
Seems should be: 
(step 4 in "Setup Hadoop Environment" section on page https://eagle.incubator.apache.org/docs/quick-start.html)

"Instruction #4 ..." does not point to the position clearly.